src/common/converters/data_decoded.rs

```diff
@@ -120,6 +123,27 @@ impl DataDecoded {
             Some(index)
         }
     }
+
+    pub fn has_nested_delegated(&self) -> bool {
+        if self.method == MULTI_SEND {
+            if let Some(value_decoded_type) =
```

Comment:
If was thinking if we should be more defensive and check for all parameters (on the root) if they have a `valueDecoded` which corresponds to `InternalTransaction`. We should then make sure that this is only checked when `trusted_for_delegate_call == true`. .... But maybe I am too paranoid :grimacing:

Reply:
And also remove the check on the `method` ... as we should probably check this for all methods ... again not sure if I am too paranoid

Reply:
I don't think this is a blocker right now, maybe something to play around with on a ecosystem friday.